tests/regression/run_regression.py

```diff
@@ -59,7 +58,7 @@
             diffs.append(
                 [
                     filename,
-                    ("Rows differ between reference and export:\n" f"{compared}"),
+                    ("Rows differ between reference and export:\n", f"{compared}"),
```

Comment:
This change doesn't seem right - I think the original intention of this parens and code was to have two lines of string concatenation, like:

```
                    (
                        "Rows differ between reference and export:\n"
                        f"{compared}")
                    )
```

Like the block above has. But at some point it must have got moved to the same line.

With the change in this PR, you'd be printing a tuple instead of a string.

Reply:
moving to a single string

this happened a lot - this is the thing that has me nervous atm. I don't have a test to test if the regression test runner is working correctly, which is why i didn't catch this one. feels a little like turtles all the way down, but can add a tech debt ticket for it.